styles/styles.css

```diff
@@ -0,0 +1,582 @@
+ul,
+li {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Work Sans', sans-serif;
+  font-size: .8rem;
+  line-height: 1.4rem;
+}
+
+h1 {
+  font-family: 'Neuton', sans-serif;
+  color: #093c60;
+  font-size: 2rem;
+}
+
+h2 {
+  font-family: 'Neuton', sans-serif;
+  color: #093c60;
+  font-size: 1.4rem;
+  margin: 0;
+}
+
+h3 {
+  font-family: 'Neuton', sans-serif;
+  color: #093c60;
+  font-size: 1.2rem;
+  margin: 0;
+}
+
+h4,
+h5,
+h6 {
+  font-family: 'Neuton', sans-serif;
+  color: #093c60;
+  margin: 0;
+}
+
+
+/* skip link */
+
+.hidden {
+  position: relative;
+  left: -10000px;
+  top: auto;
+}
+
+.hidden:focus {
+  left: 0;
+}
+
+
+/* header */
+
+header {
+  background-image: url("../images/header-image.jpg");
+  background-size: cover;
+  padding: 2rem 6rem 6rem;
+}
+
+.header-headline {
+  color: #81b741;
+  font-style: italic;
+  line-height: .2rem;
+  margin: 0;
+}
+
+.flex-header {
+  display: flex;
+  height: 17rem;
+  flex-wrap: wrap;
+}
+
+.flex-header-1 {
+  flex: 1;
+  align-items: flex-start;
+}
+
+.flex-header-1 img {
+  width: 18.5rem;
+}
+
+.flex-header-2 {
```

Comment:
be careful naming classes like this. It's hard to read. What is 1? what is 2? Maybe something like flex-header-logo-container and flex-header-checkout ? Something like that.